doc/fix_dt.txt

```diff
@@ -0,0 +1,85 @@
+vsq = fix column for y component of velocity squared, prefaced by "f_" :l
+wsq = fix column for z component of velocity squared, prefaced by "f_" :l
+mode = {warn} or {use/calculated/dt} :l
+extra = {store/cell/dt} (optional) :l
+:ule
+
+[Examples:]
+
+fix DT dt 500 0.1 0.25 0.1 c_lambda f_1\[2\] f_1\[3\] f_1\[4\] f_1\[5\] warn
+fix DT dt 500 0.1 0.25 0.1 f_lambda f_1\[2\] f_1\[3\] f_1\[4\] f_1\[5\] use/calculated/dt store/cell/dt :pre
+
+These commands will calculate a global timestep every 500 steps, either 1) using the user-provided timestep
+and warning if it is greater than the calculated timestep or 2) using the calculated timestep and
+ignoring the user-provided value.
+
+[Description:]
+
+Calculate a global timestep based on cell mean collision and transit times. The user-suppled timestep is used prior to the
+first invocation of this fix. Each time this fix is invoked, a candidate timestep is calculated in each cell, and then both an
+average candidate timestep (dt_avg) and a minimum candidate timestep (dt_min) are computed over all cells.  The global timestep
+is then determined using the formula:
+
+dt_global = (1-avg_dt_weight)*dt_min + avg_dt_weight*dt_avg :pre
+
+In the candidate timestep calculation for each cell, the mean collision time is multiplied by the user-supplied collision_fraction
+described above to compute a timestep candidate.  Then for each coordinate direction, the cell size in that direction is
+divided by the velocity magnitude in that direction and then multiplied by the user-supplied transit_fraction described above to
+compute a timestep candidate.  The cell timestep is set to be the minimum of these candidate timesteps.  Collision and
+transit fractions in the interval (0,1) simply provide a user-defined safety margin for the collision time and transit time
+estimates.  Values greater than one enable the user to increase the timestep above the recommended timestep by any desired factor.
+
+If this fix ID is used as a keyword in the "stats_style"_stats_style.html command, the calculated global timestep is printed.  A zero value
+indicates that the fix has not yet calculated a global timestep and that the user-suppled timestep is being used.  If the fix
+has been invoked, the last calculated value of the timestep will be displayed.  If the mode is set to {warn}, the calculated timestep will be
+displayed but will not be used to advance the simulation.  Note that the {dt} keyword in the stats_style command will display the actual
+timestep being used in the simulation.  The {time} option in the stats_style command prints the current simulation time.
+
+[Example:]
+
+stats_style     step time dt f_DT cpu np :pre
+
+:line
+
+[Restart, output info:]
+
+The global time and the current global timestep are written to "binary restart files"_restart.html.  This breaks backward compatibility
+with older restart files, even if this fix is not invoked.
```

Comment:
@akstagg can you remove this note about backward compatibility in the docs--we will post this in the patch release notes.